app/main.py

```diff
@@ -2,0 +1,20 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError(
+            "Invalid command format. Use: mv source_file destination_path"
+        )
+
+    source_file, destination_path = parts[1], parts[2]
+
+    if not os.path.isfile(source_file):
+        raise FileNotFoundError(f"No such file: '{source_file}'")
+
+    dest_dir = os.path.dirname(destination_path)
```

Comment:
Consider using `os.path.join` for constructing paths to ensure cross-platform compatibility, as recommended in the checklist.